frontend/server/src/Controllers/User.php

```diff
@@ -4650,6 +4651,61 @@ public static function apiRevokeAPIToken(\OmegaUp\Request $r) {
         \OmegaUp\DAO\APITokens::deleteByName($r->user->user_id, $name);
         return ['status' => 'ok'];
     }
+
+    /**
+     * @return array{entrypoint: string, templateProperties: array{payload: VerificationParentalTokenDetailsPayload, title: \OmegaUp\TranslationString}}
+     *
+     * @omegaup-request-param string $parental_verification_token
+     */
+    public static function getVerificationParentalTokenDetailsForTypeScript(
+        \OmegaUp\Request $r
+    ): array {
+        $r->ensureIdentity();
+
+        $token = $r->ensureString(
+            'parental_verification_token',
+            validator: fn (string $token) => preg_match(
+                '/^[a-zA-Z0-9]{25}$/',
+                $token
+            ) === 1
+        );
+        $hasParentalVerificationToken = false;
+
+        $userId = \OmegaUp\DAO\Users::findByParentalToken($token);
```

Comment:
Here you are getting the whole User object, not only `userId`
```suggestion
        $user = \OmegaUp\DAO\Users::findByParentalToken($token);
```